media/Jsontracer/index.html

```diff
@@ -49,11 +49,19 @@
 -->
 <head>
   <meta charset="utf-8">
+  <!--
+    Use a content security policy to only allow loading images from https or from our extension directory,
```

Comment:
Is there any case that we loads images from https?

Reply:
we don't load images. so i will change `img-src 'none'` and comment